src/lib/components/wallets/Wallets.svelte

```diff
@@ -116,9 +133,11 @@
         </div>
     </div>
 {:else}
-    {#each filteredWallets as wallet}
-        <Wallet wallet={wallet} on:connect={handleWalletConnect} />
-    {/each}
+    {#await orderedWallets then value}
+        {#each value as wallet}
```

Comment:
you can tell that the name `value` is not nice because it says `each value as wallet`, perhaps it'd be better if it said `each wallets as wallet` instead